recipes/smbprotocol/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "smbprotocol" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 98218617ce75870b5b0ea3951269e5db169b1e86509d09d61b64c0a749a3a1f3
+
+build:
+  number: 0
```

Can this be `noarch: python`?
```suggestion
  noarch: python
  number: 0
```

I guess it doesn't matter if the `python-gssapi >=1.4.1` constraint is also applied to Windows?

---

No, that should be fine. It's only run constrained and the earliest version of `python-gssapi` on CF is 1.6.9 😄

---

I don't actually have permissions to push to this branch but I'm happy for you @chrisburr to commit that change, or if it's easier I opened a new PR with that change: https://github.com/conda-forge/staged-recipes/pull/12481

---

CI passed over in https://github.com/conda-forge/staged-recipes/pull/12481 so I'm fine with either being merged!

---

Thanks!